Training 441 Crawl/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<!--
+    All Rights Reserved
+    This software is proprietary information of
+    Intelligent Sense
+    Use is subject to license terms.
+    Filename: index.html
+-->
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel='stylesheet' href='https://use.fontawesome.com/releases/v5.7.0/css/all.css'>
+    <title>Redes Sociales</title>
+</head>
+<body>
+    
+    <div class="main-container">
+        <h1 class="title">Redes Sociales</h1>
+        <div class="facebook box" >
+            <p class="facebook--font">Facebook es una herramienta social que conecta a la gente con sus
+                amigosy otras personas que trabajan, estudian y viven en su 
+                entorno
+            </p>
+            <div class="facebook__icon">
+                <a href="http://www.facebook.com">
+                    <i class="fab fa-facebook-square fa-3x "></i>
+                </a>
+            </div>
+        </div>
+        <div class="twitter box">
+            <p class="twitter--font">
+                Twitter es un servicio de microblogging. La red permite enviar
+                de texto plano de corta longitud con un máximo de 140 caracteres,
+                llamados tweets, que se muestran en la página principal del 
+                usuario.
+                <div class="twitter__icon">
```

Comment:
probably, not so important. But here you can simplify your html by putting the class twitter__icon in the <a> tag and removing the div element

Reply:
The same applies to the others